### ISSUE TITLE
matcher `changeEtherBalances` breaks with ethers 5.0.25

### ISSUE THREAD

**Issue by clifton** [EthWorks/Waffle#421]
```typescript
    const amount = parseEther('0.05');
    await expect(() =>
      faucet.connect(wallet).withdraw(amount)
    ).to.changeEtherBalances([faucet, wallet], [amount.mul(-1), amount]);
```

```
(node:778233) UnhandledPromiseRejectionWarning: TypeError: account.getAddress is not a function
    at Object.getAddressOf (node_modules/@ethereum-waffle/chai/dist/cjs/matchers/misc/account.js:13:24)
```

reverting back to ethers 5.0.24 fixes the problem. happy to create this issue in ethers if its appropriate

**Comment by vanruch** [comment]
Hi @clifton, thanks for reporting!

I couldn't reproduce the bug myself. Could you check the types of `faucet` and `wallet`, please?

**Comment by clifton** [comment]
Those are contracts created by `hardhat-waffle`'s `deployContract`.  This seems to be fixed in ethers `5.0.26` without any other changes made in the app, so I'll close the issue. thanks!
